packages/tauri/src-tauri/src/main.rs

```diff
@@ -20,21 +19,7 @@ fn main() {
         .add_default_file_submenu()
         .add_default_edit_submenu()
         .add_default_window_submenu()
-        .add_default_help_submenu(),
     )
-    .on_menu_event(|event| {
-      let event_name = event.menu_item_id();
-      match event_name {
-        "get_help" => {
-          shell::open(
-            "https://padloc.app/help/".to_string(),
-            None,
-          )
-          .unwrap();
-        }
-        _ => {}
-      }
-    })
```

Comment:
Couldn't find a different way to open a url and whitelisting the `shell` api just for that didn't seem justified.